gradlew

```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
This is automatically generated by Gradle, but the heap memory seems to be too low.

Reply:
I agree with this concern, and will revert this auto-generated change. I don't know why the gradle team felt that a 64Mbyte heap was sufficient.